frontend/src/pages/Admin/Components/Broadcast/AddBroadcasts/AddBroadcasts.jsx

```diff
@@ -70,17 +84,54 @@ export function AddBroadcasts() {
       tagRef.current.value = "";
     }
   };
+  const addImageUrl = () => {
+    const url = imageRef.current.value;
+    if (url.trim()) {
+      setImageUrls(prevUrl => [...prevUrl, url.trim()]);
+      setFormData({ ...formData, imageUrl: [...formData.imageUrl, url.trim()] });
+      imageRef.current.value = "";
+    }
+  }
 
   const removeTag = tag => {
     setTags(tags.filter(t => t !== tag));
     setFormData({ ...formData, tags: formData.tags.filter(t => t !== tag) });
   };
-
+  const removeImageUrl = url => {
+    setImageUrls(imageUrls.filter(u => u !== url));
+    setFormData({ ...formData, imageUrl: formData.imageUrl.filter(u => u !== url) });
+  }
+  async function uploadData(formData) {
+    try {
+      let url = `${END_POINT}/broadcast`
+      const response = await fetch(url, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          Authorization: `Bearer ${localStorage.getItem('token')}`
+        },
+        body: JSON.stringify(formData)
+      });
+      const data = await response.json();
+      setIsUploadingData(false)
+      setToastMessage("broadcats added successfully!")
```

Comment:
Even after api is giving `400` as response, I am seeing this toast message.

Also, please correct the message to `broadcast added successfully!`